x/keyrotation/module.go

```diff
@@ -0,0 +1,131 @@
+package keyrotation
```

Comment:
@dzmitry-lahoda as i understand now
this PR does not contains **anymore** changes related to prefix change from `cetnauri` to `pica`
PR contains only changes to `key rotation`.
What this key rotation suppose todo according to prefix changes
Seems that we need to close this pr if we do not any key rotation and i will push changes from other branch.